wandb/sdk/internal/file_stream.py

```diff
@@ -287,33 +278,25 @@ def _thread_body(self):
                             "complete": False,
                             "failed": False,
                             "dropped": self._dropped_chunks,
-                            "uploaded": list(uploaded),
+                            "uploaded": list(self.uploaded),
                         },
                     )
                 )
-                uploaded = set()
-        # post the final close message. (item is self.Finish instance now)
-        request_with_retry(
-            self._client.post,
-            self._endpoint,
-            json={
-                "complete": True,
-                "exitcode": int(finished.exitcode),
-                "dropped": self._dropped_chunks,
-                "uploaded": list(uploaded),
-            },
-        )
+                self.uploaded = set()
 
-    def _thread_except_body(self):
-        # TODO: Consolidate with internal_util.ExceptionThread
-        try:
-            self._thread_body()
-        except Exception as e:
-            exc_info = sys.exc_info()
-            self._exc_info = exc_info
-            logger.exception("generic exception in filestream thread")
-            util.sentry_exc(exc_info, delay=True)
-            raise e
```

Comment:
Would be good to test what happens when we throw in exception.  We should confirm we're atleast getting a stacktrace in `debug-internal.log`